tracing-lib-jaeger/pom.xml

```diff
@@ -46,6 +46,12 @@
         <dependency>
             <groupId>io.jaegertracing</groupId>
             <artifactId>jaeger-client</artifactId>
+            <exclusions>
```

Comment:
Not sure this makes sense here, or if it should be excluded in clients that use this library.

I.e., you are biasing this pom because of one of our products, right?

Reply:
This pull request was based on the following:

I have two dependencies, one is jaeger-client, the second is jaeger-thrift(shaded). If I depend on the shaded jaeger-thrift the projects using this library don't need to make any changes. I can't depend on the non-shaded version of jaeger-thrift because it results in conflicts, and the users might not be able to exclude it due to it depending on a different version of what they're using.

The problem before this change is that I'm pulling the two versions of jaeger-thrift, the shaded that I explicitly depend on and the non-shaded version which is brought by jaeger-client.